contracts/Example3.sol

```diff
@@ -1,14 +1,20 @@
+// SPDX-License-Identifier: MIT
+
 pragma solidity ^0.8.4;
 
 contract Example3 {
-    uint256 marginPercentage = 30;
-    function getOwnerMargin(uint256 amount)
+    uint8 marginPercentage = 30;
+    function getOwnerMargin(uint32 amount)
```

Comment:
It would be wrong to make an assumption that amount would always fit into uint32. Here the expected optimisation is to store `marginPercentage` into a memory variable inside the function since it is being accessed twice